apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/TransactionBroadcastedContent.tsx

```diff
@@ -28,26 +33,34 @@ export function TransactionBroadcastedContent(props: TransactionBroadcastedConte
         sourceCurrency={sourceCurrency?.name}
         targetCurrency={targetCurrency?.name}
         provider={provider}
-        {...swapDefaultTrack}
+        {...(swapId ? swapDefaultTrack : sellDefaultTrack)}
```

Comment:
Logic must be base in mode.
```suggestion
{...(mode === 
        mode === ExchangeModeEnum. ? swapDefaultTrack : {flow: mode)}
```